src/pypromice/L0toL1.py

```diff
@@ -137,22 +139,31 @@ def _flagNAN(ds, flag_file=None):
     ds : xr.Dataset
         Level 0 data with flagged data
     '''
-    # flag_file = "./data/flags/" + ds.attrs["station_id"] + ".csv"
-    if flag_file is None:
-        print('Flag file not given. No data flagged')
+    ds = ds_in.copy()
+    try:
+        os.mkdir('local')
+        os.mkdir('local/flags')
+        os.mkdir('local/adjustments')
+    except:
+        pass
+    flag_url = "https://raw.githubusercontent.com/GEUS-Glaciology-and-Climate/PROMICE-AWS-data-issues/master/flags/" 
+    try:
+        urllib.request.urlretrieve(flag_url + ds.attrs["station_id"] + ".csv",
+                               "local/flags/" + ds.attrs["station_id"] + ".csv")
+    except:
```

Comment:
In general, I like to have the exception be specific to the expected error that is being handled.

Reply:
Like this?
https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/861067805bfe84dc348a2838dbbe7249ae523bb0/src/pypromice/L0toL1.py#L153

Reply:
Not sure. I guess we should run it and see how/if it fails.

Reply:
New exception looks like this:
https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/d76897cedb0bb8086a68a4b57a00bb90cb4af519/src/pypromice/L0toL1.py#L166